gibbon-compiler/src/Gibbon/L0/Syntax.hs

```diff
@@ -34,7 +35,13 @@ type DDefs0   = DDefs Ty0
 type DDef0    = DDef Ty0
 type FunDef0  = FunDef Exp0
 type FunDefs0 = FunDefs Exp0
-type Prog0    = Prog Exp0
+
+type Prog0 = Prog Exp0
+
+-------------------------------------------------------------------------------
```

Comment:
Why the empty lines and the dashes? I'd just continue the list of aliases that was already there (with Ddfe0, FunDef0, etc.)